src/lib/isMobilePhone.js

```diff
@@ -123,6 +123,7 @@ const phones = {
   'lt-LT': /^(\+370|8)\d{8}$/,
   'lv-LV': /^(\+?371)2\d{7}$/,
   'mg-MG': /^((\+?261|0)(2|3)\d)?\d{7}$/,
+  "mk-Mk": /^\+389(70|71|72|75|76|77|78)\d{6}$/,
```

Comment:
You should use singlequotes
```suggestion
  'mk-Mk': /^\+389(70|71|72|75|76|77|78)\d{6}$/,
```

Reply:
Thanks for your suggestion! The test cases have passed now. 😊